### PR TITLE
RHDEVDOCS-3922 - metrics page UI improvements

### PR DESCRIPTION
**Pull request by bburt-rh**
Summary: This PR updates 2 existing procedures to add documentation for the Duplicate Query feature and autocomplete suggestions added to the PromQL Query Browser on the Observe > Metrics page in the OCP web console.

This PR also moves some "Additional resources" xref content that was incorrectly placed in proc modules and puts this content into the correct place in the assembly file.

- Aligned team: DevTools
- For branches: 4.11
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3922
- Direct link to doc preview (RH VPN access required): 
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3922-observe-metrics-page-improvements/monitoring/managing-metrics.html#querying-metrics-for-all-projects-as-an-administrator_managing-metrics 
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3922-observe-metrics-page-improvements/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics
- SME review:  @jgbernalp 
- QE review: @Tai-RedHat 
- Peer review: @ tbd

### CODE REVIEW

File: modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc [hunk 1]
```diff
@@ -29,14 +29,14 @@ Developers can only use the *Developer* perspective and not the *Administrator*
 
 . Select the project that you want to view metrics for in the *Project:* list.
 
-. Choose a query from the *Select Query* list, or run a custom PromQL query by selecting *Show PromQL*.
+. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by selecting *Show PromQL*.
+
+. Optional: Select *Custom query* from the *Select query* list to enter a new query. 
+As you type, autocomplete suggestions appear in a drop-down list.
+These suggestions include functions and metrics.
+You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression.
```
Comment (jgbernalp) [inline modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc:37]: In the developer perspective keyboard arrows don't work when selecting a suggestion because we are still using the old component. There is a task for it: https://issues.redhat.com/browse/OU-46

Comment (bburt-rh) [reply modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc:37]: Thanks. I've removed that sentence and added `Click a suggested item to select it.`
`

**Comment by jgbernalp** [comment]
LGTM

File: modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc [hunk 2]
```diff
@@ -26,9 +21,19 @@ Only cluster administrators have access to the third-party UIs provided with {pr
...
+You can also move your mouse pointer over a suggested item to view a brief description of that item.
+====
 
 . To add multiple queries, select *Add Query*.
 
+. To duplicate an existing query, select {kebab} next to the query, then choose *Duplicate query*.
+
 . To delete a query, select {kebab} next to the query, then choose *Delete query*.
 
 . To disable a query from being run, select {kebab} next to the query and choose *Disable query*.
```
Comment (jc-berger) [inline modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc:39]: In the preview, I see steps 2 & 8. I know you didn't write those, so no worries if you don't want to make a change to them.  But right now, they're the only steps that start with a verb instead of an infinitive phrase, so it reads kind of awkward. For consistency, I would rephrase them to:

2 To view a list of predefined queries, select **Insert Metrics at Cursor**. 

8 To run the queries that you have created, run **Run Queries**. 


Again, I know this isn't content you originally wrote, so it's fine to leave it as is :)

File: modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc [hunk 3]
```diff
@@ -29,14 +29,14 @@ Developers can only use the *Developer* perspective and not the *Administrator*
 
 . Select the project that you want to view metrics for in the *Project:* list.
```
Comment (jc-berger) [inline modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc:30]: In the preview, I see step 1. I think how it is written reads well. If you want all the steps to be consistent and start with the verb, `Select` you could rephrase step 1 to:

1. Select **Observe -> Metrics** from the **Developer** perspective in the OpenShift Container Platform web console. 

Note, I think users will not be thrown from this "inconsistency" so it's fine to leave as is. As a writer, the sentence structure difference in one step does kind of throw me, haha.

File: monitoring/managing-metrics.adoc [hunk 4]
```diff
@@ -34,19 +34,27 @@ include::modules/monitoring-specifying-how-a-service-is-monitored.adoc[leveloffs
 include::modules/monitoring-querying-metrics.adoc[leveloffset=+1]
 //include::modules/monitoring-contents-of-the-metrics-ui.adoc[leveloffset=+2]
 include::modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation] for more information about creating PromQL queries.
```
Comment (jc-berger) [inline monitoring/managing-metrics.adoc:41]: For readability and scanability, I think it reads better to have the link at the end of the sentence. Makes it easier for users to click the link, in my opinion. 

`For more information about creating PromQL queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation]`

File: monitoring/managing-metrics.adoc [hunk 5]
```diff
@@ -34,19 +34,27 @@ include::modules/monitoring-specifying-how-a-service-is-monitored.adoc[leveloffs
...
+
 include::modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
 
-* See the xref:../monitoring/managing-metrics.adoc#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer] for details on accessing non-cluster metrics as a developer or a privileged user
+* See the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation] for more information about creating PromQL queries.
```
Comment (jc-berger) [inline monitoring/managing-metrics.adoc:48]: Same comment as above, consider putting the link at the end of the sentence. 

`For more information about creating PromQL queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation].`

File: monitoring/managing-metrics.adoc [hunk 6]
```diff
@@ -34,19 +34,27 @@ include::modules/monitoring-specifying-how-a-service-is-monitored.adoc[leveloffs
...
 
 include::modules/monitoring-exploring-the-visualized-metrics.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
 
-* See the xref:../monitoring/managing-metrics.adoc#querying-metrics_managing-metrics[Querying metrics] section on using the PromQL interface
+* See xref:../monitoring/managing-metrics.adoc#querying-metrics_managing-metrics[Querying metrics] for details on using the PromQL interface
```
Comment (jc-berger) [inline monitoring/managing-metrics.adoc:55]: Same comments as above, I think it's easier to scan and click if the links are at the end of the sentence. 

However, the change is not needed :)

**Review by jc-berger — COMMENTED** [review]
I consulted the openshift checklist list when conducting this peer review: https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.mgz8v4w1kg86

File: monitoring/managing-metrics.adoc (hunk 5 above)
Comment (bburt-rh) [reply monitoring/managing-metrics.adoc:48]: This was existing content that I moved, but I agree it reads better with your edits. I'll make the changes

File: monitoring/managing-metrics.adoc (hunk 6 above)
Comment (bburt-rh) [reply monitoring/managing-metrics.adoc:55]: This was existing content that I moved, but I agree it reads better with your edits. I'll make the changes

File: modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc (hunk 3 above)
Comment (bburt-rh) [reply modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc:30]: Good point. I'll follow your suggestion for better consistency.

File: modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc (hunk 2 above)
Comment (bburt-rh) [reply modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc:39]: I revised the final step of the existing content for better consistency with the other steps.

**Review by jc-berger — APPROVED** [review]
/lgtm
